components/ws-manager/pkg/manager/create.go

```diff
@@ -569,6 +624,20 @@ func (m *Manager) createWorkspaceContainer(startContext *startWorkspaceContext)
 
 	image := fmt.Sprintf("%s/%s/%s", m.Config.RegistryFacadeHost, regapi.ProviderPrefixRemote, startContext.Request.Id)
 
+	volMounts := []corev1.VolumeMount{
+		{
+			Name:             workspaceVolumeName,
+			MountPath:        workspaceDir,
+			ReadOnly:         false,
+			MountPropagation: &mountPropagation,
+		},
+		{
+			MountPath:        "/.workspace",
+			Name:             "daemon-mount",
+			MountPropagation: &mountPropagation,
+		},
+	}
```

Comment:
nit: why move this to a variable beforehand and not leave it inline?

Reply:
bad refactor on my part, was cleaning up after all my changes and forgot to undo this one.